client/app/controllers/collection/add.js

```diff
@@ -66,20 +66,40 @@ export default Ember.Controller.extend({
             info: null
         }));
         this.saveParameter = this.saveParameter.bind(this);
+        this.closeSection = this.closeSection.bind(this);
+        this.openSection = this.openSection.bind(this);
+        this.create_widget = this.create_widget.bind(this);
```

Comment:
This I think would get covered in the ticket I made to adhere to airbnb styles, OC-77